Extending-OSVR/ConfiguringHDKViveTracking.md

```diff
@@ -0,0 +1,260 @@
+            "count": 14
+        }
+    },
+    "semantic": {
+		"hmd": {
+            "$target": "tracker/0",
+			"button": "button/0",
+            "proximity": "button/1"
+		},
+		"puck": {
+            "$target": "tracker/3"
+		},
+        "ipd": "analog/0",
+        "controller": {
+            "left": {
+                "$target": "tracker/1",
+                "system": "button/2",
+                "menu": "button/3",
+                "grip": "button/4",
+                "trackpad": {
+                    "x": "analog/1",
+                    "y": "analog/2",
+                    "touch": "button/5",
+                    "button": "button/6"
+                },
+                "trigger": {
+                    "$target": "analog/3",
+                    "button": "button/7"
+                }
+            },
+            "right": {
+                "$target": "tracker/2",
+                "system": "button/8",
+                "menu": "button/9",
+                "grip": "button/10",
+                "trackpad": {
+                    "x": "analog/4",
+                    "y": "analog/5",
+                    "touch": "button/11",
+                    "button": "button/12"
+                },
+                "trigger": {
+                    "$target": "analog/6",
+                    "button": "button/13"
+                }
+            }
+        }
+    },
+    "automaticAliases": {
+        "/me/head": "semantic/hmd",
+        "/me/puck": "semantic/puck",
+        "/me/ipd": "semantic/ipd",
+        "/controller": "semantic/controller/*",
+        "/me/hands/left": "semantic/controller/left",
+        "/me/hands/right": "semantic/controller/right"
+    }
+}
+````
+
+Take the time to review the device descriptor to understand how the Vive devices are factored into OSVR interfaces. The parts related to the puck are the assignment to tracker 3, and the automatic alias:
+````json
+"/me/puck": "semantic/puck",
+````
+
+### OSVR Server Configuration
+Normally for HDK2 with orientation-only tracking, we’d use a config file such as [/sample-configs/osvr_server_config.renderManager.HDKv2.0.direct.json](https://github.com/OSVR/OSVR-Core/blob/master/apps/sample-configs/osvr_server_config.renderManager.HDKv2.0.direct.json). 
+
+Take a look at the last line in the config file above. The syntax is similar to how we’ll use the tracking puck:
+````json
+"aliases": {"/me/head":"/com_osvr_Multiserver/OSVRHackerDevKit0/semantic/hmd"}
+````
+Instead of setting up an alias to the Multiserver plugin, we want to access the Vive plugin. We saw earlier that there is an automatic alias in the Vive device descriptor named “/me/puck”. Our new config file with puck tracking for the head is:
+````json
+"aliases": {"/me/head":"/me/puck”}
+````
+That’s it! We are now able to move the puck around to change the image in our display. We probably have a little extra work to do depending on the puck's expected orientation and how it is mounted to the HDK.
```

Comment:
You might explain that any OSVR clients requesting poses from `/me/head` will get poses from `/me/puck` which gets poses from `/com_osvr_Vive/semantic/puck` (or whatever) which gets it from... Essentially, following all the aliases down the chain so they can see how it all connects.